setup.py

```diff
@@ -90,6 +90,9 @@ def run(self):
           'absl-py>=0.1.2',
           'google-api-python-client>=1.1',
       ],
+      extra_require=[
+          'memory_profiler',
+      ],
```

Comment:
This needs to be a dict with the name of the option and then a list of the requirements.
i.e.
```python
extra_require={'profiler' : ['memory_profiler']}
```